src/components/Slider/index.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useEffect } from 'react';
+
+import classnames from '../../utils/classNames';
+import { IProps } from './types';
+
+import './index.scss';
+
+const baseClass = 'vant-slider';
+
+const Slider = ({
+  range = { min: 0, max: 100 },
+  size = { width: 400, height: 5 },
```

Comment:
the value here should have unit, like `400px`

Reply:
fixed